tests/success-associated-origin-backwards-compat.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Is there a consumer of this tests somewhere? And/or why is it necessary to have a backwards compatibility test?

Reply:
This is something we need to add in Chromium. The backwards compat check makes sure that any feeds that still have associated origins still work.